frontend/lib/ui/Feedback/Feedback.tsx

```diff
@@ -30,7 +30,9 @@ export function Feedback({ id, type, data, apiError, children }: FeedbackProps)
         <div className="feedback-item">
           <header>
             {renderIconForType(type)}
-            <h3>Sorry, er ging iets mis</h3>
+            <h3 id="feedback-title-0" tabIndex={-1}>
```

Comment:
you can't put fixed ID's in ui elements, they are not guaranteed to be used only once on a page.
so it has to be a prop.

Reply:
Very good point! Changed in d1bf0ed431654f0ec14291f58a46a924fc3242c0